docs/rfcs/0042-evaluation-for-updated-features.md

```diff
@@ -0,0 +1,120 @@
+# Evaluation for updated FeatureFlag
+
+
+## Proposal
+
+- This feature evaluates only those feature flags that have been updated since the last evaluation.
+  - Computational efficiency will be improved because the number of the feature flags to be evaluated will be reduced.
+- We can make the tag optional.
+  - Previously, specifying a tag was required when executing GetEvaluations to reduce the response size.
+  - But, with this proposal, the response size can be kept small, so specifying a tag can be made optional.
+
+
+
+
+## Implementation
+
+The SDK stores the timestamp of the evaluation's execution.
+When GetEvaluations is called again, the timestamp is sent to the server as a request parameter.
+
+The server checks it against the updatedAt value of the feature flags, and only evaluates those that have been updated since the previous evaluation.
+
+As an exception, the following feature flags must be evaluated regardless of the timestamp value.
+- Feature flags that depend on the feature flags that need to be evaluated
+- (if user attributes have been updated) Feature flags with targeting rules
+
+If user attributes have been updated, the result of the evaluation may also change.
+Since the server has to know that update, the SDK sends the `IsUserAttributesChanged` flag to the server as a request parameter like the timestamp.
+
+In addition, the following changes are required in both the server and SDK implementations:
+- Since only the updated feature flags are evaluated and returned, the SDK's implementation for updating local data needs to be changed.
+- The server needs to return information about archived feature flags to the SDK.
+  - So the server must put the archived feature flags to Redis.
+- The validation of the request parameter `Tag` needs to be removed.
+
+
+### Evaluation on Server
```

Comment:
Can you write a description of how to evaluate feature flags currently for feature contributors?